### PR TITLE
App/zigbee2mqtt and App/eclipse-mosquitto

### PR DESCRIPTION
**Pull request by tommy-hartmann**
I added [zigbee2mqtt 🐝](https://www.zigbee2mqtt.io/) and [eclipse mosquitto](https://mosquitto.org/) as the message broker. For mosquitto i added the cedalo manangement center so that the broker can be configured via web UI.

I tested both within my own setup on a synology NAS.

## Summary by CodeRabbit

- **New Features**
    - Updated `mosquitto.conf` file with configuration settings for Mosquitto v2.0, including default port, security plugin, and RBAC rules.
    - Added `dynsec-setup.sh` script for setting up default admin credentials in Dynamic Security Plugin configurations.
    - Introduced UI for managing access to Eclipse Mosquitto MQTT Broker.

### CODE REVIEW

File: apps/eclipse-mosquitto/data/config/mosquitto.conf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+# Mosquitto v2.0 Configuration File
+
+# Default Port
+listener 1883
+
+# Dynamic Security Plugin
+plugin /usr/lib/mosquitto_dynamic_security.so
```
[the remark this answers is not in the document]
Comment (tommy-hartmann) [reply apps/eclipse-mosquitto/data/config/mosquitto.conf:20]: the plugin is available in the docker image:

```
$docker exec -it <container_id> /bin/sh
$ls -al /usr/lib/mosquitto_dynamic_security.so 
-rwxr-xr-x    1 root     root        112320 Jan 27 00:50 /usr/lib/mosquitto_dynamic_security.so
```

File: apps/eclipse-mosquitto/data/config/mosquitto.conf [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# Dynamic Security Plugin
+plugin /usr/lib/mosquitto_dynamic_security.so 
+
+# Path to the Dynamic Security JSON file
+# WILL BE GENERATED THROUGH INIT CONTAINER
+plugin_opt_config_file /mosquitto/config/dynamic-security.json
+
+# Ever User / Client should follow the given RBAC rules
+per_listener_settings false
```
[the remark this answers is not in the document]
Comment (tommy-hartmann) [reply apps/eclipse-mosquitto/data/config/mosquitto.conf:27]: this is the default setting for the management center app and it works fine

File: apps/eclipse-mosquitto/data/scripts/dynsec-setup.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+# Setup Script to be executed in a Docker Init Container
+
+# Set Default Admin Credentials for Dynamic Security Plugin Configuration
+DEFAULT_DYNSEC_ADMIN=admin
+DEFAULT_DYNSEC_PASSWORD=securePassword
```
[the remark this answers is not in the document]
Comment (tommy-hartmann) [reply apps/eclipse-mosquitto/data/scripts/dynsec-setup.sh:19]: this is overwritten by an environment variable set in the docker-compose.yml file in the next lines.
This only applies if the environment variable is not availe.

File: apps/eclipse-mosquitto/data/scripts/dynsec-setup.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+DYNSEC_FILE_PATH=/mosquitto/config/dynamic-security.json
+
+# Set values if provided via Environment Variables in the Docker Init Container
+MQTT_DYNSEC_ADMIN_USER=${MQTT_DYNSEC_ADMIN_USER:-DEFAULT_DYNSEC_ADMIN}
+MQTT_DYNSEC_ADMIN_PASSWORD=${MQTT_DYNSEC_ADMIN_PASSWORD:-DEFAULT_DYNSEC_PASSWORD}
```
[the remark this answers is not in the document]
Comment (tommy-hartmann) [reply apps/eclipse-mosquitto/data/scripts/dynsec-setup.sh:24]: then i have to define the default varibales in the field section of config.json as hidden fields, right?

File: apps/eclipse-mosquitto/data/scripts/dynsec-setup.sh [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+echo "Admin/Pass: ${MQTT_DYNSEC_ADMIN_USER}/${MQTT_DYNSEC_ADMIN_PASSWORD}" ## DEBUG
```
[the remark this answers is not in the document]
Comment (tommy-hartmann) [reply apps/eclipse-mosquitto/data/scripts/dynsec-setup.sh]: resolved within 3822234bd4c0f91b836089ccc418862d0fb3e26d

**Comment by tommy-hartmann** [comment]
I also found an issue for the request of the eclipse mosquitto broker: 
[[APP REQUEST] Mosquito #2000](https://github.com/runtipi/runtipi-appstore/issues/2000)

**Comment by meienberger** [comment]
Thank you @tommy-hartmann

**Pull request merged**